### DISCUSSION TITLE
Skip uploading node_modules folder

### DISCUSSION THREAD

**Discussion by matthewjrallen** [iterate-ch/cyberduck#16127] [Q&A]
I've added node_modules to the uploads filter but it still uploads the folder. The folder is within another folder. Is this the correct way of adding it?

```
.*~\..*|\.DS_Store|\.svn|CVS|\.git|\.gitignore|\.gitattributes|\.bzr|\.bzrignore|\.bzrtags|\.hg|\.hgignore|\.hgtags|node_modules|\.vs_code
```

**Comment by tatze96** [comment]
This should exclude the folder correctly. Could you post the complete property line so I can check for typos or other parts that could prevent it from working correctly?

**Reply by matthewjrallen** [reply]
I've tried today and it's working now for node_modules but it has still uploaded the .vs_code folder and .gitignore file. Do I need the backslash on them. What does the backslash do?
